### PR TITLE
Added documentation for canRowSelect prop.

### PR DESCRIPTION
**Pull request by WiXSL**

### CODE REVIEW

File: docs/List.md [hunk 1]
```diff
@@ -906,6 +907,20 @@ const PostList = props => (
     </List>
 )
 ```
+
+### `canRowSelect`
+
+You can customize which rows will show a selection checkbox using the `canRowSelect` prop. It expects a function that will receive the record of each `<DatagridRow>` and returns a boolean expression.  For instance, this code shows a checkbox only for rows with an id grather than 300:
```
Comment (fzaninotto) [inline docs/List.md]:
```suggestion
You can customize which rows will show a selection checkbox using the `canRowSelect` prop. It expects a function that will receive the record of each `<DatagridRow>` and returns a boolean expression.  For instance, this code shows a checkbox only for rows with an id greather than 300:
```

File: docs/List.md [hunk 2]
```diff
@@ -906,6 +907,20 @@ const PostList = props => (
     </List>
 )
 ```
+
+### `canRowSelect`
+
+You can customize which rows will show a selection checkbox using the `canRowSelect` prop. It expects a function that will receive the record of each `<DatagridRow>` and returns a boolean expression.  For instance, this code shows a checkbox only for rows with an id greather than 300:
```
Comment (fzaninotto) [inline docs/List.md]:
```suggestion
You can customize which rows will show a selection checkbox using the `canRowSelect` prop. It expects a function that will receive the record of each `<DatagridRow>` and returns a boolean expression.  For instance, this code shows a checkbox only for rows with an id greater than 300:
```

**Comment by WiXSL** [comment]
Done!
I change it to `isRowSelectable`.

**Comment by fzaninotto** [comment]
Thanks!

**Pull request merged**